pandas/core/dtypes/common.py

```diff
@@ -1934,7 +1934,7 @@ def pandas_dtype(dtype):
             except TypeError:
                 pass
 
-        elif dtype.startswith('interval[') or dtype.startswith('Interval['):
+        elif dtype.startswith('interval') or dtype.startswith('Interval'):
```

Comment:
Changed this because switching to `pandas_dtype` caused a test to break since it was passing `'interval'` as the dtype, which appears to be valid:
```python
In [1]: from pandas.core.dtypes.common import is_interval_dtype

In [2]: is_interval_dtype('interval')
Out[2]: True
```